uievents/mouse/mousemove_prevent_default_action.tentative.html

```diff
@@ -58,9 +58,11 @@
 
     assert_equals(event_log.toString(), expected_events.toString(),
         "received events");
+    document.removeEventListener("selectionchange", logEvents);
```

Comment:
I suggest removing the listener on test cleanup (like [this](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/pointerevents/pointerevent_after_target_removed.html;drc=5f2e13565e42396168ae26246204ed7ee06b1277;l=71)), otherwise any assert failure above will leave the listener intact.

Reply:
Oh yeah, this is exactly what I was looking for. Thanks for the pointer

Reply:
Fixed in https://github.com/web-platform-tests/wpt/pull/42471/commits/aa9357ac906c0a9bdb257a48a585bac24f001017